### PR TITLE
Fix comments from Hristo PR

### PR DESCRIPTION
**Pull request by 2HEADSs**
fixed comments and replace checkDatesAndTime function

### CODE REVIEW

File: src/utils/checkDatesAndTime.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function checkDatesAndTime(dates) {
+    for (let i = 0; i < dates.length; i++) {
+        const currentDate = dates[i].date;
+        const startTime = dates[i].startTime;
+        const endTime = dates[i].endTime;
+        if (startTime >= endTime) {
+            throw new Error("Start-time can't be after or equal to end-time!");
+        }
+        for (let j = 1; j < dates.length; j++) {
```
[the remark this answers is not in the document]
Comment (2HEADSs) [reply src/utils/checkDatesAndTime.js:9]: yes it works it is tested

File: src/controllers/eventController.js [hunk 2]
```diff
@@ -81,15 +94,17 @@ eventController.get('/:id', async (req, res) => {
 eventController.put('/:id', async (req, res) => {
     try {
         if (Object.keys(req.body).length === 0) {
-            throw new Error('Invalid request Body!')
+            throw new Error('Invalid request Body!');
         }
-        
+        if (req.body.dates[0].startTime >= req.body.dates[0].endTime) {
```
[the remark this answers is not in the document]
Comment (2HEADSs) [reply src/controllers/eventController.js]: fixed 
also fixed not captured bug

File: src/controllers/eventController.js [hunk 3]
```diff
@@ -96,14 +96,11 @@ eventController.put('/:id', async (req, res) => {
...
             throw new Error('Invalid request Body!');
         }
-        if (req.body.dates[0].startTime >= req.body.dates[0].endTime) {
-            throw new Error("Start-time can't be after or equal to end-time!");
-        }
-
+        checkDatesAndTime(req.body.dates)
         const event = await findEventByID(req.params.id);
-
+        const createor = event?.creator._id.toLocaleString()
```
Comment (Getsov) [inline src/controllers/eventController.js:101]: creatorId *

Comment (2HEADSs) [reply src/controllers/eventController.js]: fixed

**Review by Getsov — APPROVED** [review]

**Pull request merged**